packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -126,6 +126,14 @@
             </component>
             <!-- END ERROR BOUNDARY -->
 
+            <!-- Offers -->
+            <icon-text
+                v-if="hasOffer"
+                data-test-id="restaurant-offer"
+                :text="offer"
+                :is-bold="true">
```

Comment:
No need to update but you can shorthand props with a value of `true` like this one to `is-bold`.

Reply:
I had no idea you could do that! thanks, I'll update as it's neat!